modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -171,17 +173,22 @@ export default function EditNotificationTemplate({
 						objectDefinitionId,
 						subject,
 						to,
-					})
+					});
+					setTitle(name);
+				}
 			);
 		}
+		else {
+			setTitle(Liferay.Language.get('untitled-notification-template'));
+		}
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [notificationTemplateId]);
 
 	return (
 		<ClayForm onSubmit={handleSubmit}>
 			<ClayManagementToolbar className="lfr__notification-template-management-tollbar">
 				<ClayManagementToolbar.ItemList>
-					<h2>{Liferay.Language.get('notification-templates')}</h2>
+					<h2>{title}</h2>
```

Comment:
Hey @MrJohn1911 As we modified the `useState` above, we can use it here to check if the `templateTitle` exists and place it, otherwise we use the language key from before using [Nullish Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) like:

```tsx
{templateTitle ??
	Liferay.Language.get('notification-templates')}
```